### ISSUE TITLE
Database invalid version

### ISSUE THREAD

**Issue by lmrodriguezr** [chklovski/CheckM2#31]
The database download seems to be failing to determine the downloaded version.

I'm using v1.0.0 (as modified by #30 to allow installation) and Python 3.8.15.

This reproduces the error:

```
$> checkm2 database --download
[12/18/2022 04:34:36 PM] INFO: Command: Download database. Checking internal path information.
[12/18/2022 04:34:38 PM] INFO: Downloading https://zenodo.org/api/files/fd3bc532-cd84-4907-b078-2e05a1e46803/checkm2_database.tar.gz to /Users/miguel/databases/checkm2_database.tar.gz.
100%|███████████████████████████████████████████████████████████████████████████████████████| 1.74G/1.74G [08:00<00:00, 3.61MiB/s]
[12/18/2022 04:42:40 PM] INFO: Extracting files from archive...
[12/18/2022 04:43:11 PM] INFO: Verifying version and checksums...
[12/18/2022 04:43:11 PM] INFO: Verification success.
Traceback (most recent call last):
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/bin/checkm2", line 4, in <module>
    __import__('pkg_resources').run_script('CheckM2==1.0.0', 'checkm2')
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/pkg_resources/__init__.py", line 672, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1472, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/CheckM2-1.0.0-py3.8.egg/EGG-INFO/scripts/checkm2", line 244, in <module>
    fileManager.DiamondDB().download_database(args.path)
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/CheckM2-1.0.0-py3.8.egg/checkm2/fileManager.py", line 140, in download_database
    if versionControl.VersionControl().checksum_version_validate_DIAMOND():
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/CheckM2-1.0.0-py3.8.egg/checkm2/versionControl.py", line 119, in checksum_version_validate_DIAMOND
    return self.__validateVersion(self.version, cutoff_version)
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/CheckM2-1.0.0-py3.8.egg/checkm2/versionControl.py", line 58, in __validateVersion
    return v_compare.parse(str(query)) >= v_compare.parse(str(ref))
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/packaging/version.py", line 52, in parse
    return Version(version)
  File "/usr/local/Caskroom/miniconda/base/envs/checkm2/lib/python3.8/site-packages/packaging/version.py", line 197, in __init__
    raise InvalidVersion(f"Invalid version: '{version}'")
packaging.version.InvalidVersion: Invalid version: '10    0.0.0
14    0.1.0
Name: incompatible_below_checkm2ver, dtype: object'
```

**Comment by xinehc** [comment]
I got the same error after updating to 1.0.0. However `testrun` can still produce some results.

Can we ignore this error ATM or is this error database related so we need to wait until it being fixed?

**Comment by GabeAl** [comment]
Same issue here.

**Comment by chklovski** [comment]
Hi, the error seems to be due to some sort of issue in parsing CheckM2's current version and comparing it against the expected database requirements. It's safe to ignore it if you are on v 1.0.0 and if testrun completes successfully. Will look into fixing it shortly.

**Comment by chklovski** [comment]
Hi, what output does running checkm2 `--version` produce?

**Comment by chklovski** [comment]
Sorry - what does the command ` conda info | grep setuptools` show?

**Comment by chklovski** [comment]
Sorry, never mind. Turns out it was an artifact of a bad pandas parsing code. Fixed in release [1.0.1](https://github.com/chklovski/CheckM2/releases/tag/1.0.1).
